inc/member-directory.php

```diff
@@ -0,0 +1,336 @@
+			'sanitization_cb' => 'absint', // custom sanitization callback parameter
+			'escape_cb'       => 'absint',  // custom escaping callback parameter
+		) );
+
+		$member_info->add_field( array(
+			'name'       => __( 'INN Member Since', 'cmb2' ),
+			'desc'       => __( '', 'cmb2' ),
+			'id'         => $prefix . 'inn_join_year',
+			'type'       => 'text_small',
+			'show_on_cb' => 'cmb2_hide_if_no_cats', // function should return a bool value
+			'sanitization_cb' => 'absint', // custom sanitization callback parameter
+			'escape_cb'       => 'absint',  // custom escaping callback parameter
+		) );
+
+		// Email text field
+		$member_info->add_field( array(
+			'name' => __( 'Contact Email', 'inn' ),
+			'desc' => __( '', 'cmb2' ),
+			'id'   => $prefix . 'email',
+			'type' => 'text_email',
+		) );
+
+		$member_info->add_field( array(
+			'name' => __( 'Website URL', 'inn' ),
+			'desc' => __( '', 'cmb2' ),
+			'id'   => $prefix . 'url',
+			'type' => 'text_url',
+			'protocols' => array( 'http', 'https' ), // Array of allowed protocols
+		) );
+
+		$member_info->add_field( array(
+			'name' => __( 'Donate URL', 'inn' ),
+			'desc' => __( '', 'cmb2' ),
+			'id'   => $prefix . 'donate_url',
+			'type' => 'text_url',
+			'protocols' => array( 'http', 'https' ), // Array of allowed protocols
+		) );
+
+		$member_info->add_field( array(
+			'name' => __( 'RSS Feed', 'inn' ),
+			'desc' => __( '', 'cmb2' ),
+			'id'   => $prefix . 'rss_feed',
+			'type' => 'text_url',
+			'protocols' => array( 'http', 'https' ), // Array of allowed protocols
+		) );
+
+		$member_info->add_field( array(
+			'name' => __( 'Twitter Profile', 'inn' ),
+			'desc' => __( '', 'cmb2' ),
+			'id'   => $prefix . 'twitter_url',
+			'type' => 'text_url',
+			'protocols' => array( 'http', 'https' ), // Array of allowed protocols
+		) );
+
+		$member_info->add_field( array(
+			'name' => __( 'Facebook Page', 'inn' ),
+			'desc' => __( '', 'cmb2' ),
+			'id'   => $prefix . 'facebook_url',
+			'type' => 'text_url',
+			'protocols' => array( 'http', 'https' ), // Array of allowed protocols
+		) );
+
+		$member_info->add_field( array(
+			'name' => __( 'Youtube URL', 'inn' ),
+			'desc' => __( '', 'cmb2' ),
+			'id'   => $prefix . 'youtube_url',
+			'type' => 'text_url',
+			'protocols' => array( 'http', 'https' ), // Array of allowed protocols
+		) );
+
+		$member_info->add_field( array(
+			'name' => __( 'Google+ URL', 'inn' ),
+			'desc' => __( '', 'cmb2' ),
+			'id'   => $prefix . 'google_plus_url',
+			'type' => 'text_url',
+			'protocols' => array( 'http', 'https' ), // Array of allowed protocols
+		) );
+
+		$member_info->add_field( array(
+			'name' => __( 'Contact Phone', 'inn' ),
+			'desc' => __( '', 'inn' ),
+			'id'   => $prefix . 'phone_number',
+			'type' => 'text_small',
+		) );
+
+		$member_info->add_field( array(
+			'name' => __( 'Address', 'inn' ),
+			'desc' => __( '', 'inn' ),
+			'id'   => $prefix . 'address',
+			'type' => 'address',
```

Comment:
Testing with the current master branch of this theme and the wp.org version of CMB2, the 'address' box doesn't appear to be displaying an editable field.
<img width="636" alt="screen shot 2017-05-30 at 10 50 44 am" src="https://cloud.githubusercontent.com/assets/1754187/26589857/22c4607c-4527-11e7-84c6-f2462597ec64.png">
<img width="507" alt="screen shot 2017-05-30 at 10 50 47 am" src="https://cloud.githubusercontent.com/assets/1754187/26589856/22c3a0e2-4527-11e7-8c0a-7b5fa9894891.png">

This is true on existing INN members and on the new-member-creation page.